plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -137,15 +145,36 @@ class NetworkingFormSection extends mixin(StoreMixin) {
       address = `${hostname}${Networking.L4LB_ADDRESS}${port}`;
     }
 
+    const loadBalancerTooltipContent = (
```

Comment:
This looks like the same message used elsewhere. I'm wondering if all these tooltips should be extracted to a single file and all the help texts into another. That would make it easier to update the docs links too